client/my-sites/email/email-management/index.jsx

```diff
@@ -137,6 +137,19 @@ class EmailManagement extends React.Component {
 				secondaryAction: translate( 'Add Email Forwarding' ),
 				secondaryActionURL: emailManagementForwarding( selectedSiteSlug, selectedDomainName ),
 			};
+			if ( isMappedDomain( selectedDomain ) ) {
+				Object.assign( emptyContentProps, {
+					title: translate( 'G Suite is not supported on this domain' ),
```

Comment:
Why do we override this `title` property with the same value already present in `emptyContentProps`?